packages/components/src/ui/ariakit-tooltip/index.tsx

```diff
@@ -80,6 +81,7 @@ function AriaToolTip( props: ToolTipProps ) {
 							shortcut={ shortcut }
 						/>
 					) }
+					<TooltipArrow size={ 0 } />
```

Comment:
The `placement` of tooltip was no longer working after recent updates, with the following error in unit tests: 

```
TypeError: Cannot destructure property 'name' of '_ref' as it is undefined.

      at node_modules/@floating-ui/core/dist/floating-ui.core.umd.js:110:11
          at Array.filter (<anonymous>)
      at Object.computePosition (node_modules/@floating-ui/core/dist/floating-ui.core.umd.js:108:22)
      at updatePosition2 (node_modules/@ariakit/react-core/cjs/__chunks/P3RGUF6X.cjs:228:21)
      at update (node_modules/@ariakit/react-core/cjs/__chunks/P3RGUF6X.cjs:257:11)
```

This issue was resolved in `floating-ui/core@1.0.2`, but we use `1.0.1`, the required minimum by `floating-ui/dom`. Currently, we have `floating-ui/react-dom` pinned, and [we're manually updating](https://github.com/WordPress/gutenberg/pull/48402#issuecomment-1621484780) the `package-lock.json` for releases. So I wonder if we can update this without causing issues or accidental overrides in the future. 

**Specifically for tooltip**, the error above occurs when the `ArrowElement` is `null`. If we pass in the `TooltipArrow` component, this resolves the issue. 

I was thinking because of the [component stores](https://ariakit.org/guide/component-stores), that we might be able to set the arrow to an empty `HTMLElement` or mock it, as there are the methods `setState` and `setArrowElement`, but I'm not sure of the best approach there. 

A simple workaround would be setting the arrow size to 0, which avoids `null` and is hidden visually. I _think_ it would be ok in this case since the arrow is set to `aria-hidden`, and the size wouldn't be exposed in our API for changes to be made. Or at least until we decided to add the arrow in the future, which I think would be ideal; but, for now, it isn't a feature of the current tooltip we are replacing. 

Thoughts? @mirka